### ISSUE TITLE
brackets v1.12 file not found

### ISSUE THREAD

**Issue by ydesgagn** [caskroom/homebrew-cask#43473]
#### General troubleshooting steps

- [x] I have retried my command with `--force` and the issue is still present.
- [x] I have checked the instructions for [reporting bugs](https://github.com/caskroom/homebrew-cask#reporting-bugs) (or [making requests](https://github.com/caskroom/homebrew-cask#requests)) before opening the issue.
  - [x] None of the templates was appropriate for my issue, or I’m not sure.
- [x] I ran `brew update-reset && brew update` and retried my command.
- [x] I ran `brew doctor`, fixed as many issues as possible and retried my command.
- [x] I understand that [if I ignore these instructions, my issue may be closed without review](https://github.com/caskroom/homebrew-cask/blob/master/doc/faq/closing_issues_without_review.md).

#### Description of issue

brew cask install brackets end up with a file not found.

A commit was made on this cask 2 days ago.

#### Output of your command with `--verbose --debug`

```
brew cask install brackets --verbose --debug
==> Hbc::Installer#install
==> Printing caveats
==> Hbc::Installer#fetch
==> Satisfying dependencies
==> Downloading
==> Downloading https://github.com/adobe/brackets/releases/download/release-1.12/Brackets.Release.1.12.dmg
/usr/bin/curl --show-error --user-agent Homebrew/1.5.2-37-g701ecb4 (Macintosh; Intel Mac OS X 10.13.3) curl/7.54.0 --fail --location --remote-time --continue-at - --output /Users/yves/Library/Caches/Homebrew/Cask/brackets--1.12.dmg.incomplete https://github.com/adobe/brackets/releases/download/release-1.12/Brackets.Release.1.12.dmg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
curl: (22) The requested URL returned error: 404 Not Found
Error: Download failed on Cask 'brackets' with message: Download failed: https://github.com/adobe/brackets/releases/download/release-1.12/Brackets.Release.1.12.dmg
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/download.rb:42:in `rescue in fetch'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/download.rb:40:in `fetch'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/download.rb:15:in `perform'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/installer.rb:126:in `download'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/installer.rb:51:in `fetch'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/installer.rb:75:in `install'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/install.rb:20:in `block in run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/install.rb:14:in `each'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/install.rb:14:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli/abstract_command.rb:35:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:98:in `run_command'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:168:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:132:in `run'
/usr/local/Homebrew/Library/Homebrew/cmd/cask.rb:8:in `cask'
/usr/local/Homebrew/Library/Homebrew/brew.rb:100:in `<main>'
Error: Kernel.exit
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:173:in `exit'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:173:in `rescue in run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:156:in `run'
/usr/local/Homebrew/Library/Homebrew/cask/lib/hbc/cli.rb:132:in `run'
/usr/local/Homebrew/Library/Homebrew/cmd/cask.rb:8:in `cask'
/usr/local/Homebrew/Library/Homebrew/brew.rb:100:in `<main>'
```

#### Output of `brew cask doctor`

```
brew cask doctor
==> Homebrew-Cask Version
Homebrew-Cask 1.5.2-37-g701ecb4
caskroom/homebrew-cask (git revision aa612; last commit 2018-02-01)
==> macOS
10.13.3
==> SIP
Enabled
==> Java
9.0.4, 1.8.0_162
==> Homebrew-Cask Install Location
<NONE>
==> Homebrew-Cask Staging Location
/usr/local/Caskroom
==> Homebrew-Cask Cached Downloads
~/Library/Caches/Homebrew/Cask (3 files, 420.9MB)
==> Homebrew-Cask Taps:
/usr/local/Homebrew/Library/Taps/caskroom/homebrew-cask (3912 casks)
/usr/local/Homebrew/Library/Taps/caskroom/homebrew-fonts (1159 casks)
/usr/local/Homebrew/Library/Taps/caskroom/homebrew-versions (177 casks)
==> Contents of $LOAD_PATH
/usr/local/Homebrew/Library/Homebrew/cask/lib
/usr/local/Homebrew/Library/Homebrew
/Library/Ruby/Gems/2.3.0/gems/did_you_mean-1.0.0/lib
/Library/Ruby/Site/2.3.0
/Library/Ruby/Site/2.3.0/x86_64-darwin17
/Library/Ruby/Site/2.3.0/universal-darwin17
/Library/Ruby/Site
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/vendor_ruby/2.3.0
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/vendor_ruby/2.3.0/x86_64-darwin17
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/vendor_ruby/2.3.0/universal-darwin17
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/vendor_ruby
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/x86_64-darwin17
/System/Library/Frameworks/Ruby.framework/Versions/2.3/usr/lib/ruby/2.3.0/universal-darwin17
==> Environment Variables
LC_ALL="en_US.UTF-8"
PATH="/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/Homebrew/Library/Homebrew/shims/scm"
SHELL="/usr/local/bin/bash"
```

**Comment by brianmorton** [comment]
@ydesgagn you should be able to revert to the version from before the update with
```
brew cask uninstall brackets
brew cask install https://raw.githubusercontent.com/caskroom/homebrew-cask/5b1cea1558d579160500d56beae136e1eeb3a7c9/Casks/brackets.rb
```
(note I have not tested the above)

**Comment by commitay** [comment]
https://github.com/adobe/brackets/releases/tag/release-1.12

>  We are pulling down Brackets 1.12 release assets for a brief period to analyse LOC issues reported by users on various platforms. Rest assured this will be fixed on priority and re-establish the 1.12 download links again.

https://github.com/caskroom/homebrew-cask/pull/43482
